marbl/marbl.go

```diff
@@ -75,6 +76,7 @@ func (s *Stream) loop() {
 }
 
 func (s *Stream) Close() error {
+	s.closec <- struct{}{}
```

Comment:
What does this do?  Removing it doesn't cause any tests to fail, and the next line is already signaling that the stream is closed, right?